src/harness/fourslash.ts

```diff
@@ -949,6 +949,22 @@ namespace FourSlash {
             this.verifySymbol(symbol, declarationRanges);
         }
 
+        public symbolsInScope(range: Range): ts.Symbol[] {
+            const node = this.goToAndGetNode(range);
+            return this.getChecker().getSymbolsInScope(node, ts.SymbolFlags.Value | ts.SymbolFlags.Type | ts.SymbolFlags.Namespace);

```

Comment:
Not currently; `All` wouldn't be a good name since that's only 3 of 27 flags, but this is often useful; I'm tempted to just name it `ValueTypeNamespace`.